lib/min_heap.rb

```diff
@@ -14,18 +14,23 @@ def initialize
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O(log n), where n is the number of items
+  # Space Complexity: O(n) 
   def add(key, value = key)
```

Comment:
👍 , although the add method is just O(log n) as the call stack will depend on the number of nodes.  You're not making n recursive calls.